src/plat/qemu-riscv-virt/config.cmake

```diff
@@ -0,0 +1,103 @@
+#
+# Copyright 2022, HENSOLDT Cyber
+# Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+#
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+cmake_minimum_required(VERSION 3.7.2)
+
+declare_platform(qemu-riscv-virt KernelPlatformQEMURiscVVirt PLAT_QEMU_RISCV_VIRT KernelArchRiscV)
+
+if(KernelPlatformQEMURiscVVirt)
+
+    if("${KernelSel4Arch}" STREQUAL riscv64)
+        declare_seL4_arch(riscv64)
+    elseif("${KernelSel4Arch}" STREQUAL riscv32)
+        # This is still untested
+        declare_seL4_arch(riscv32)
+    else()
+        fallback_declare_seL4_arch_default(riscv64)
+    endif()
+
+    config_set(KernelOpenSBIPlatform OPENSBI_PLATFORM "generic")
+    config_set(KernelPlatformFirstHartID FIRST_HART_ID 1)
```

Comment:
The Spike uses 0 as the first hart ID, is there a reason the QEMU one is 1? Just curious, I couldn't get the RISC-V VMM working without changing this to 0.

Reply:
Yes, you are right. Thanks. I've been testing with 2 cores, thus this was working for me.